draft-ietf-quic-http.md

```diff
@@ -362,8 +362,8 @@ stream of the dependent request, not the data stream.
 
 HTTP/QUIC supports server push as described in {{!RFC7540}}. During connection
 establishment, the client indicates whether it is willing to receive server
-pushes via the SETTINGS_DISABLE_PUSH setting in the SETTINGS frame (see
-{{connection-establishment}}), which defaults to 1 (true).
+pushes via the SETTINGS_ENABLE_PUSH setting in the SETTINGS frame (see
+{{connection-establishment}}), which defaults to 0 (false).
```

Comment:
I would remove the value and just say ", which is disabled by default".  The way that we express Boolean values is somewhat special and suggesting a concrete value invites misinterpretation.